packages/api-derive/src/accounts/types.ts

```diff
@@ -18,6 +18,7 @@ export interface DeriveAccountRegistration {
   matrix?: string | undefined;
   other?: Record<string, string> | undefined;
   parent?: AccountId | undefined;
+  subs?: AccountId[] | undefined;
```

Comment:
Is the actual type `Vec<AccountId>` returned from `subsResponse[1];` or `AccountId[]`?